amida-tech/amida-messaging-microservice
# [DEVOPS-365] Mild changes to ensure that the deployment works as

#### What's this PR do?
Slightly tweaks the process to allow for correct deployments in Docker, and therefore production.

#### Related JIRA tickets:
DEVOPS-365

#### How should this be manually tested?
First, create a network:
`docker network create indaba-network`

Ensure that the docker instance has a database to work with. We're assuming the username and password in your `.env` are both set to `amida`, if not then please change. To start a docker instance of a database, run:
`docker run --name messaging-db --network indaba-network -e POSTGRES_USER=amida -e POSTGRES_PASSWORD=amida -e POSTGRES_DB=amida-messaging-microservice postgres:9.6`

Then run a build in Docker with:
 `docker build -t amida-messaging-microservice .`

Finally try a deployment: 
`docker run -p 4001:4001 --name amida-messaging-microservice -v <Absolute path to your...>\.env:/app/dist/.env -e MESSAGING_SERVICE_PG_HOST=messaging-db amida-messaging-microservice`

If you have any Postman requests, fire them at the endpoint to confirm completion. 

#### Any background context you want to provide?
This blocks an ease-of-deployment for Indaba. I am currently trying to get four instances running for four people as simply as possible to cut down on... hour drain. 

#### Screenshots (if appropriate):
